README.md

```diff
@@ -88,6 +88,10 @@ Table of Contents
     - [**Foxy Droid**](https://github.com/kitsunyan/foxy-droid) <sup>**[[F-Droid](https://f-droid.org/app/nya.kitsunyan.foxydroid)]**</sup>
 - [**Aurora Store**](https://gitlab.com/AuroraOSS/AuroraStore) <sup>**[[F-Droid](https://f-droid.org/app/com.aurora.store)]**</sup>
 - [**Yalp Store**](https://github.com/yeriomin/YalpStore) <sup>**[[F-Droid](https://f-droid.org/app/com.github.yeriomin.yalpstore)]**</sup>
+- [**Sky Droid**](https://github.com/redsolver/skydroid)
+- [**Appteka**](https://github.com/solkin/appteka-android)
+- [**IzzyOnDroid**](https://gitlab.com/sunilpaulmathew/izzyondroid) <sup>**[[F-Droid](https://f-droid.org/app/in.sunilpaulmathew.izzyondroid)]**</sup>
+- [**NetHunter Store**](https://gitlab.com/kalilinux/nethunter/store/nethunter-store-client)
```

Comment:
This seems rather inactive, no activity since ~ 2 years.

Reply:
Sky Droid - active project, no issues

Appteka - It is an alternative app store where apps shared by its users, though I personally don't like the app (see discussion [here](https://gitlab.com/IzzyOnDroid/repo/-/issues/235)). But well it's an OSS & ppl should know about its existence.

IzzyOnDroid - It is like any other F-droid client but focused on only one repo. Plus it promotes izzy's work :)

NetHunter Store - It's an app/repo from Kali Linux community https://store.nethunter.com/ . Though I agree the app is not that active but project is not abandoned. But you decide._(btw YallpStore also seems inactive, last update- oct,2018)_

Reply:
Sorry for the confusion, my comment was only related to hethunter store.
But let's keep it for now.